### ISSUE TITLE
Content Review/Audit v1

### ISSUE THREAD

**Issue by diegocob** [GSA/px-bears-drupal#83]

**Comment by EnriqueOrt** [comment]
Need to be reorganized and broken down into tasks. More complex than originally planned for. Potential sync with Scott

**Comment by diegocob** [comment]
Will add the EN content for v2 home, application and other pages

**Comment by diegocob** [comment]
Meeting with Gene and Scott to define the requirements

**Comment by diegocob** [comment]
Agreed with Gene and Scott this is mostly for content, since it will not be used by devs. Waiting for USAGov folks to edit the new benefits content, I will close it and copy to the excel format of the data bank.

**Comment by diegocob** [comment]
Benefits draft and content pages draft submitted for revision. Once I hear back will incllude comments and be ready for inputting to Drupal.
